### PR TITLE
added loren epson

### PR DESCRIPTION
**Pull request by SergiusHp**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,190 +1,215 @@
...
+                        flex-direction: column;
+                    }
+                    
+                    .midle_block {
+                        display: flex;
+                        flex-direction: column;
+                        align-items: center;
+                        width: 70%;
+                        height: 100%;
+                        /* background-color: rgb(77, 108, 245); */
```
Comment (lapc002) [inline style.css]: please remove comment

File: style.css [hunk 2]
```diff
@@ -1,190 +1,215 @@
...
+                    .color-icon {
+                        color: rgb(30, 119, 202);
+                    }
+                    
+                    .foter_block_end_img {
+                        display: flex;
+                        background-image: url(./jpg/mini_2.png);
+                        width: 30px;
+                        height: 30px;
+                    }
```
Comment (lapc002) [inline style.css]: no new line in the end of file . please add it

File: style.css (hunk 1 above)
Comment (SergiusHp) [reply style.css]: done

File: style.css (hunk 2 above)
Comment (SergiusHp) [reply style.css]: done

**Pull request merged**